dev/new-rule/new-rule-template.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const Rule = require('./base');
+const AstNodeInfo = require('../helpers/ast-node-info');
+
+// *** Change template to 'Real Error Message to Report' ***
+const ERROR_MESSAGE = 'Error Message to Report';
```

Comment:
Could you add `TODO:` prefixes in comments or messages that the developer will need to update? Using a common prefix like that makes it easier to search and recognize places that need to be updated (some IDEs even highlight TODOs).